src/content/blog/working-with-vite-in-ddev.md

```diff
@@ -59,7 +59,25 @@ In order to use Vite in our DDEV projects, we need to do two things:
     })
     ```
 
-Some more customizations might be needed depending on your CMS / framework, see [List of PHP CMS integrations](#list-of-php-cmsintegrations) below.
+    Please make sure your project runs on `https://` and mkcert was executed once as describe in [DDEV installation docs](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/).
```

Comment:
I don't think we need to mention `mkcert` here.  DDEV setup _usually_ configures everything correctly and this might confuses new DDEV developers who "don't remember installing mkcert".

Maybe something like the below?

```suggestion
    This guide assumes your project runs on `https://`. If you can not access the HTTPS version of your project, please see [DDEV installation docs](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/).
```